### DISCUSSION TITLE
2024 norns with infinite pots: usability feedback

### DISCUSSION THREAD

**Discussion by dndrks** [monome/norns#1775] [General]
we received our first bit of feedback in the support inbox re: the new infinite pots, so i wanted to start a discussion to track dialogue.

> I love the feel, I am finding however the speed of adjusting parameters is so wildly different than my older norns I'm having troubles adjusting. Is there any way to adjust the settings on how fast the new potentiometers roll through things?

*nb. i actually feel like the pots move **slower** than i'd like for wide-range parameters -- maybe we should consider button-hold combos to do "coarse" and "fine-tune" adjustment?*

**Comment by dndrks** [comment]
i think it's safe to call this closed via https://github.com/monome/norns/releases/tag/v2.8.3 <333
amazing speed @tehn , thank you!!!

**Comment by hallmar** [comment]
If it wouldn't be too much a pain, it would maybe be nice to re-open this again? 

I feel that quick-sweeps of values still isn't all the way there, with the encoders you can sweep pretty fast but with the pots it takes around 6 times the movement to do the same. 

I also posted on the Lines release thread, might be wrong to post in both places but I wanted to document this here also

**Comment by tehn** [comment]
I can keep tuning the system, thanks the feedback.

**Reply by hallmar** [reply]
Great, thanks!! 

Maybe another magic number will do the trick
